From 694cf8936459e22bf8b1f28676973ef5a32a9a68 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Thu, 5 Jul 2012 20:42:59 +0300
Subject: [PATCH 228/370] confc.c: define c2_confc{,_ctx}_bob as static

c2_confc{,_ctx}_bob are not supposed to be used outside of confc.c.

RB: r/865/diff/1/?file=22712#file22712line371
---
 conf/confc.c | 4 ++--
 conf/confc.h | 2 --
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/conf/confc.c b/conf/confc.c
index a2fb7a3..afd4430 100644
--- a/conf/confc.c
+++ b/conf/confc.c
@@ -369,7 +369,7 @@ static bool confc_invariant(const struct c2_confc *confc)
 }
 
 /* XXX TODO: .bt_check = confc_invariant */
-struct c2_bob_type c2_confc_bob = XXX;
+static struct c2_bob_type c2_confc_bob = XXX;
 
 static bool confc_is_initialized(const struct c2_confc *confc)
 {
@@ -385,7 +385,7 @@ static void on_replied(struct c2_rpc_item *item);
 static bool on_object_updated(struct c2_clink *link);
 
 /* XXX TODO: .bt_check = ctx_invariant */
-struct c2_bob_type c2_confc_ctx_bob = XXX;
+static struct c2_bob_type c2_confc_ctx_bob = XXX;
 
 /** States of c2_confc_ctx::fc_mach. */
 enum confc_ctx_state { S_INITIAL, S_CHECK, S_WAIT_REPLY, S_WAIT_STATUS,
diff --git a/conf/confc.h b/conf/confc.h
index dc22fb3..e62ef40 100644
--- a/conf/confc.h
+++ b/conf/confc.h
@@ -368,7 +368,6 @@ struct c2_confc {
 	 */
 	uint32_t            cc_nr_ctx;
 };
-extern struct c2_bob_type c2_confc_bob;
 
 /**
  * Initializes configuration client.
@@ -447,7 +446,6 @@ struct c2_confc_ctx {
 	/** Magic value. */
 	uint64_t                 fc_magix;
 };
-extern struct c2_bob_type c2_confc_ctx_bob;
 
 /**
  * Initializes configuration retrieval context.
-- 
1.8.3.2

